blackjax/diagnostics.py

```diff
@@ -0,0 +1,164 @@
+"""MCMC diagnoistics."""
+from typing import Union
+
+import jax
+import jax.numpy as jnp
+import numpy as np
+from scipy.fftpack import next_fast_len  # type: ignore
+
+Array = Union[np.ndarray, jnp.DeviceArray]
+
+__all__ = ["potential_scale_reduction", "effective_sample_size"]
+
+
+def potential_scale_reduction(
```

Comment:
So if I want to compute the Rhat value in a problem with 2 variables, do I have to use `tree_multimap` to compute the values?

Reply:
Use `tree_map` but yes the function takes arrays as input